<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartQuizGen - Create Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#SignUp {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                      stop:0 #0F3057, stop:0.33 #4D7EA8,
                      stop:0.66 #9DC5BB, stop:1 #E6F4F1);
}

QLabel {
    color: black;
    font-weight: normal;
}

QLineEdit {
    border: 2px solid #4D7EA8;
    border-radius: 12px;
    padding: 2px 15px;
    background-color: rgba(255, 255, 255, 0.9);
    selection-background-color: #9DC5BB;
    font-size: 10px;
    color: #333333;
}

QPushButton#FinishSignupButton {
    background-color: #0F3057;
    color: white;
    border-radius: 15px;
    font-weight: bold;
    padding: 8px 16px;
    border: none;
}

QPushButton#FinishSignupButton:hover {
    background-color: #4D7EA8;
}

QPushButton#FinishSignupButton:pressed {
    background-color: #9DC5BB;
}

QPushButton#SignInButton {
    background-color: transparent;
    color: white;
    text-decoration: underline;
    border: none;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.7);
}

QPushButton#SignInButton:hover {
    color: #9DC5BB;
}

QCheckBox {
    color: black;
    font-weight: normal;
    spacing: 8px;
    text-shadow: 0px 0px 1px rgba(255, 255, 255, 0.8);
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    border: 2px solid white;
}

QCheckBox::indicator:checked {
    background-color: #9DC5BB;
    border: 2px solid #9DC5BB;
}

QPushButton#BackHomePage {
    background-color: rgba(255, 255, 255, 0.2);
    border: none;
    border-radius: 20px;
}

QPushButton#BackHomePage:hover {
    background-color: rgba(255, 255, 255, 0.3);
}

QPushButton#SeeHidePassword, QPushButton#SeeHideConfirmPassword {
    background-color: transparent;
    border: none;
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QPushButton" name="BackHomePage">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/images/back.png</normaloff>:/images/images/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="AppLogoLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>SmartQuizGen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="CreateAccountLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Your Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="FirstNameLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="FirstNameLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FirstNameEntry">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter your first name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LastNameLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="LastNameLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LastNameEntry">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter your last name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="UsernameLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="UsernameLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UsernameEntry">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Choose a username</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="EmailLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="EmailLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EmailEntry">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter your email address</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="PasswordLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="PasswordLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="PasswordEntryLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="PasswordEntry">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Choose a strong password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SeeHidePassword">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/images/hide.png</normaloff>:/images/images/hide.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ConfirmPasswordLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="ConfirmPasswordLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirm Password</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ConfirmPasswordEntryLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="ConfirmPasswordEntry">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Re-enter your password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SeeHideConfirmPassword">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/images/hide.png</normaloff>:/images/images/hide.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Agreements">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>I agree to Terms of Service and Privacy Policy</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="FinishSignupButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="SignInButton">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Already have an account? Sign In</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
